src/plugins/intel_cpu/tests/unit/shape_inference_test/custom_shape_infer.cpp

```diff
@@ -0,0 +1,226 @@
+// Copyright (C) 2018-2023 Intel Corporation
+// SPDX-License-Identifier: Apache-2.0
+//
+#include "openvino/core/partial_shape.hpp"
+#include "openvino/core/type.hpp"
+#include <ngraph/opsets/opset1.hpp>
+#include "openvino/op/parameter.hpp"
+#include "utils/custom_shape_inference/reshape.hpp"
+#include "utils/custom_shape_inference/gather.hpp"
+#include "utils/custom_shape_inference/transpose.hpp"
+#include "utils/custom_shape_inference/color_convert.hpp"
+#include "utils/custom_shape_inference/eltwise.hpp"
+#include "utils/custom_shape_inference/adaptive_pooling.hpp"
+#include "utils/custom_shape_inference/fullyconnected.hpp"
+#include "utils/custom_shape_inference/matmul.hpp"
+#include "utils/custom_shape_inference/ngram.hpp"
+#include "utils/custom_shape_inference/one_hot.hpp"
+#include "utils/custom_shape_inference/priorbox.hpp"
+#include "utils/custom_shape_inference/priorbox_clustered.hpp"
+#include "utils/custom_shape_inference/shapeof.hpp"
+#include "utils/custom_shape_inference/strided_slice.hpp"
+// #include "utils/custom_shape_inference/deconv.hpp"
+// #include "utils/custom_shape_inference/subgraph.hpp"
+#include "ie_ngraph_utils.hpp"
+#include "custom_shape_infer.hpp"
+#include "utils/shape_inference/shape_inference_status.hpp"
+#include <gtest/gtest.h>
+namespace ov {
+namespace intel_cpu {
+namespace unit_test {
+namespace {
+#define INTEL_CPU_CUSTOM_SHAPE_INFER(__prim, __type) \
+    registerNodeIfRequired(intel_cpu, __prim, __type, __prim)
+
+class EltwiseShapeInferTestFactory : public node::EltwiseShapeInferFactory {
+public:
+    EltwiseShapeInferTestFactory(std::shared_ptr<ov::Node> op) : EltwiseShapeInferFactory() {}
+};
+
+class ShapeOfShapeInferTestFactory : public node::ShapeOfShapeInferFactory {
+public:
+    ShapeOfShapeInferTestFactory(std::shared_ptr<ov::Node> op) : ShapeOfShapeInferFactory() {}
+};
+
+class CustomShapeInferFF : public openvino::cc::Factory<Type, ShapeInferFactory*(const std::shared_ptr<ov::Node>& op)> {
+public:
+    CustomShapeInferFF():Factory("CpuCustomShapeInferTestFactory") {
+    // INTEL_CPU_CUSTOM_SHAPE_INFER(Generic, Type::Generic);
+    // INTEL_CPU_CUSTOM_SHAPE_INFER(CumSum, Type::CumSum);
```

Comment:
Do we really need all this commented code?

Reply:
ok，I will remove them